azavea/loam
# Release/1.0.0

## Overview

Bump version number and CHANGELOG for 1.0.0 release.


### Demo

Optional. Screenshots, `curl` examples, etc.


### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.


## Testing Instructions

 * How to test this PR
 * Prefer bulleted description
 * Start after checking out this branch
 * Include any setup required, such as bundling scripts, restarting services, etc.
 * Include test case and expected output


## Checklist


Resolves #XXX
